### PR TITLE
remove content pages and redirect to website instead

### PR DESCRIPTION
**Pull request by jbibla**
duplicated and outdated content pages can just redirect to lunie website

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f1e371da27f043c0f9ff9ea--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
I love it! Looks really so much better like this.

Only missing the `Careers` page though, it seems:

image

File: app/src/routes.js [hunk 1]
```diff
@@ -207,23 +207,33 @@ export default (store) => {
...
-      component: () => import(`./components/common/PageAbout`),
+      beforeEnter() {
+        location.href = `https://lunie.io/about`
+      },
     },
     {
       path: `/careers`,
-      component: () => import(`./components/common/PageCareers`),
+      beforeEnter() {
+        location.href = `https://lunie.io/careers`
```
Comment (jbibla) [inline app/src/routes.js:235]:
```suggestion
        location.href = `https://angel.co/company/lunie`
```

Comment (jbibla) [reply app/src/routes.js:235]: this ok?

Comment (Bitcoinera) [reply app/src/routes.js:235]: why don't we have the careers page in https://lunie.io as well?

Comment (faboweb) [reply app/src/routes.js:235]: I think it is ok to just use Angelist. I wouldn't mind as a developer looking for a job

**Pull request merged**